examples/libf3d/python/render-image/render_image.py

```diff
@@ -0,0 +1,21 @@
+import sys
+import f3d
+
+if __name__ == "__main__":
+    if len(sys.argv) > 4:
+        sys.exit(1)
+
+    try:
+        f3d.Engine.autoload_plugins()
+
+        eng = f3d.Engine(f3d.Window.NATIVE_OFFSCREEN)
+        eng.loader.load_geometry(sys.argv[1])
+
+        eng.window.size = 300, 300
+        img = eng.render_to_image()
```

Comment:
```suggestion
        img = eng.window.render_to_image()
```
I messed up my copy-pasting in my previous suggestion :facepalm:, `render_to_image` is a window method.